config/drupal/phpcsfixer.rules.yml

```
@@ -85,11 +85,9 @@ parameters:
                 - switch
```

should add  curly_brace_block`


```suggestion
                -  curly_brace_block
```